packages/splitter/splitter.stories.tsx

```diff
@@ -193,30 +200,139 @@ Hook.parameters = {
   }
 };
 
-export const Controlled = ({
+export const ManyHooks = ({
```

Comment:
Added a more comprehensive example to test the new fixes in Storybook and I suggest this remains for demonstrating a more complex layout.